### DISCUSSION TITLE
feat: RFP Support

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#749] [Ideas]
### What feature would you like to see?

As the title suggests, the implementation of Resist Fingerprinting (RFP) is a robust mechanism for safeguarding user privacy. In my view, integrating this feature into Zen Browser would significantly bolster its privacy protection capabilities.

Should Zen Browser already incorporate this feature, I would be genuinely delighted to learn of it :)

Best regards.

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/713_

**Comment by pricklywiggles** [comment]
Bump. I agree, this is essential.
